intelmq/bin/intelmqctl.py

```diff
@@ -1255,6 +1255,16 @@ def list_bots(self, non_zero=False, configured=False):
             val = utils.list_all_bots()
             return 0, val
 
+    def _pipeline_configuration(self):
+        pipeline_configuration = {}
+        for botid, botconfig in self.runtime_configuration.items():
+            pipeline_configuration[botid] = {"source_queue": f"{botid}-queue", "destination_queues": []}
+            if 'source_queue' in botconfig['parameters']:
```

Comment:
"global" needs to be ignored here, otherwise `intelmqctl list queues` etc. raises a KeyError

Reply:
fixed